website/source/docs/secrets/identity/index.html.md

```diff
@@ -134,6 +134,176 @@ group and has an alias that maps to the group in LDAP. If the user is removed
+| `time.now`                                                             | Current time as integral seconds since the Epoch                                        |
+| `time.now.plus.<<duration>>`                                           | Current time plus a Go-parsable [duration](https://golang.org/pkg/time/#ParseDuration)  |                             |
+| `time.now.minus.<<duration>>`                                          | Current time minus a Go-parsable [duration](https://golang.org/pkg/time/#ParseDuration) |                             |
+
+
+### Verifying Authenticity of ID Tokens Generated by Vault
+
+An identity token may be verified by the client party using the public keys
+published by Vault, or via a Vault-provided introspection endpoint.
+
+Vault will serve standard "[.well-known](https://tools.ietf.org/html/rfc5785)"
+endpoints that allow easy integration with OIDC verification libraries.
+Configuring the libraries will typically involve providing an issuer URL and
+client ID. The library will then handle key requests and can validation the
+signature and claims requirements on tokens. This approach has the advantage of
+only requiring _access_ to Vault, not _authorization_, as the .well-known
+endpoints are unauthenticated.
+
+Alternatively, the token may be sent to Vault for verification via an
+[introspection endpoint](api/secret/identity/tokens.html#introspect-a-signed-id-token).
+The response will indicate whether the token is "active" or not, as well as any
+errors that occurred during validation. Beyond simply allowing the client to
+delegate verification to Vault, using this endpoint incorporates the additional
+check of whether the entity is still active or not, which is something that
+cannot be determined from the token alone. Unlike the .well-known, accessing the
+introspection endpoint does require a valid Vault token and sufficient
+authorization.
+
+
+### Issuer Considerations
+
+The identity token system has one configurable parameter: issuer. The issuer
+`iss` claim is particularly important for proper validation of the token by
+clients, and special consideration should be given when using Identity Tokens
+with [performance replication](docs/enterprise/replication/index.html).
+Consumers of the token will request public keys from Vault using the issuer URL,
+so it much be network reachable. Furthermore, the returned keyset will include
+an issuer that must match the request.
+
+By default Vault will set the issuer to the Vault instance's
+[`api_addr`](docs/configuration/index.html#api_addr) This means that tokens
+issued in a given cluster should be validated within that same cluster.
+Alternatively, the [`issuer`](api/secret/identity/tokens.html#issuer) parameter
+may be configured explicitly. This address must point to the identity/oidc path,
```

Comment:
Is "the identity/oidc path" explicit enough?  Or should we spell it out a bit more?

Reply:
This brings up good point: should a custom issuer just be domain and port. We're appending `/v1/identity/oidc` to the api_addr, but using a provided issuer verbatim. Maybe we should append the path to both.

Reply:
I've created a task to address this (I think it should be changed), but will clarify the as-is docs now.